src/ui/popupmenu/lazy_loader.rs

```diff
@@ -19,7 +19,8 @@ struct State {
     /// Once we're loaded some (or all) data, this closure gets called if
     /// one exists. The first value in the tuple can is indication on the
     /// number of items needed before calling the closure.
-    once_loaded: Option<(Option<i32>, Box<Fn(&Vec<CompletionItemWidgetWrap>)>)>,
+    once_loaded:
+        Option<(Option<i32>, Box<dyn Fn(&Vec<CompletionItemWidgetWrap>)>)>,
```

Comment:
Was the compiler giving you warnings here? For me (rustc 1.36) doesn't complain anything.

Reply:
Yes, but only on nightly. Stable doesn’t complain, but nightly says `warning: trait objects without an explicit ‘dyn’ are deprecated` and then suggests to put a ` dyn` there. You can see it in the Travis nightly (and I think beta) checks, [see this one for master](https://travis-ci.org/vhakulinen/gnvim/jobs/567500562#L742).